### PR TITLE
add icedtea-web page

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [x] `npm test` passes
- [ ] documentation is changed or added (if applicable)
- [ ] permission has been obtained to add new logo (if applicable)
- [x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/handlebars/icedtea-web.handlebars [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<!-- TODO Style should match installation page, it doesn't quite... -->
+<main class="grey-bg upstream">
+
+  <h1 class="margin-bottom large-title">IcedTea-Web Builds</h1>
+
+  <div class="info-page-container upstream blue-link-section">
+
+    <div class="callout">
+      <h3 class="bold">What are these binaries?</h3>
+      <p>The IcedTea-Web Javaws_splash project provides a Free Software web browser plugin running applets written in the Java™ programming language and an implementation of Java™ Web Start, originally based on the NetX project.</p>
```
Comment (sxa555) [inline src/handlebars/icedtea-web.handlebars]: I'm not sure I like this wording. The main thing we want to push is the web start stuff (javaws command line) so talking about this purely in terms of the plug (which is hard to use with any modern browser now!) is probably not the best headline to use.

**Comment by SueChaplain** [comment]
@gdams - I find the placement of the page a little odd. Under the hamburger menu, we have the main AdoptOpenJDK download pages, followed by 3 sections that cover installation, migration and support. The IcedTea page is inserted under installation, which seems to break that flow. Would it be better to have a section under "Support" called "Projects" and list Iced-Tea under there. This allows for growth .... perhaps there are other projects that might want a home there too?

File: src/handlebars/icedtea-web.handlebars (hunk 1 above)
Comment (douph1) [reply src/handlebars/icedtea-web.handlebars]: Appel support will be removed soon ( in ITW 2.0 ) So it is better to not speak about.
https://github.com/AdoptOpenJDK/IcedTea-Web/pull/162
And AdoptInstaller dont register anything about applet in browser.

Comment (gdams) [reply src/handlebars/icedtea-web.handlebars]: removed

**Comment by gdams** [comment]
@SueChaplain agreed, updated

**Pull request merged**